### ISSUE TITLE
[FORMATTING] LIMIT format error

### ISSUE THREAD

**Issue by thanharrow** [sql-formatter-org/sql-formatter#412]
**Input data**
select * from account limit 10;
Indentation style: Tabular, Left
**Expected Output**

```sql
select    *
from      account
limit       10;
```

**Actual Output**

```sql
select    *
from      account
limit
          10;
```

**Usage**

- How are you calling / using the library?  I using https://sql-formatter-org.github.io/sql-formatter/
- What SQL language(s) does this apply to? SQL
- Which SQL Formatter version are you using? 10.0.0

**Comment by nene** [comment]
Thanks for reporting.

Looks like another regression from the change of how LIMIT keyword is handled. The test coverage for the other indentation styles besides the default is also fairly limited.

**Comment by nene** [comment]
The fix for this is now out in sql-formatter 10.1.0
